subscriptions/sales_flow/create_a_quotation.rst

```diff
@@ -0,0 +1,78 @@
+==============================================
+Create a quotation using subscription products
+==============================================
+
+Selling a digital product or service can give you instant gratification. However, you then have
+to work hard for a new customer to make a purchase. It costs time and money. By convincing customers
+to sign up for a subscription, you will maximize your income and streamline your cash flow. You can
+sell any type of product or service through a subscription business model.
```

Comment:
```suggestion
Selling a digital product or service gives instant gratification. However, you have
to work hard for a new customer to make a purchase. It costs time and money. 

By convincing customers to sign up for a subscription, you maximize income and 
streamline the cash flow. You can sell any type of product or service through a 
subscription business model.
```

Reply:
I made all the changes except "your income" and "your cash flow". It sounds better with "your" I think :)

Reply:
Ok :+1: